src/trainable.jl

```diff
@@ -0,0 +1,42 @@
+using .Flux: trainable
```

Comment:
This is also not needed, it seems?

Reply:
Isn't it good practice to show which function you are going to dispatch on?

Reply:
Isn't that indicated quite clearly by writing `Flux.trainable(k::ExponentialKernel) = ...` etc.? I thought the line here could just be replaced with
```suggestion
import .Flux
```